examples/node-groups/windows-node-groups/main.tf

```diff
@@ -31,10 +31,10 @@ data "aws_availability_zones" "available" {
   # Specify AZs to avoid EKS cluster creation error due to reduced capacity in an AZ.
   # Change the AZ names per capacity available in the region you selected.
   # https://aws.amazon.com/premiumsupport/knowledge-center/eks-cluster-creation-errors/
-  filter {
-    name   = "zone-name"
-    values = ["us-west-2a", "us-west-2b", "us-west-2c"]
-  }
+  # filter {
+  #   name   = "zone-name"
+  #   values = ["us-west-2a", "us-west-2b", "us-west-2c"]
```

Comment:
I believe we want to keep this filter with the hard coded AZs, similar to our approach and upcoming PR https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/424 where we are keeping hard coded region by default, same stands true for this case.

Reply:
Whoops, I missed that. I'll close this then.